servers/debugger/servers_debugger.cpp

```diff
@@ -198,7 +198,7 @@ class ServersDebugger::ScriptsProfiler : public EngineProfiler {
 	typedef ServersDebugger::ScriptFunctionInfo FunctionInfo;
 	struct ProfileInfoSort {
 		bool operator()(ScriptLanguage::ProfilingInfo *A, ScriptLanguage::ProfilingInfo *B) const {
-			return A->total_time < B->total_time;
+			return A->total_time > B->total_time;
```

Comment:
Is this only used in that one place? If not any other possible use of this part needs to be updated for the order change

Reply:
This ProfileInfoSort is only used [once, locally](https://github.com/godotengine/godot/blob/17e7f85c06366b427e5068c5b3e2940e27ff5f1d/servers/debugger/servers_debugger.cpp#L243)

The result is visible ~2 places:
1. In the order that `servers:function_signature` is [emitted](https://github.com/godotengine/godot/blob/17e7f85c06366b427e5068c5b3e2940e27ff5f1d/servers/debugger/servers_debugger.cpp#L256), but it is only consumed to put in a map [here](https://github.com/godotengine/godot/blob/17e7f85c06366b427e5068c5b3e2940e27ff5f1d/editor/debugger/script_editor_debugger.cpp#L682)
1. In the frame data [emitted](https://github.com/godotengine/godot/blob/17e7f85c06366b427e5068c5b3e2940e27ff5f1d/servers/debugger/servers_debugger.cpp#L319-L321) with `servers:profile_total` or `server:profile_frame`, [stored](https://github.com/godotengine/godot/blob/17e7f85c06366b427e5068c5b3e2940e27ff5f1d/editor/debugger/script_editor_debugger.cpp#L787) in the script funcs category.items, [added](https://github.com/godotengine/godot/blob/17e7f85c06366b427e5068c5b3e2940e27ff5f1d/editor/debugger/script_editor_debugger.cpp#L793-L795) to the UI, and consumed only [here](https://github.com/godotengine/godot/blob/90a5f23e798df8192e903fc5785e29fa093e5caa/editor/debugger/editor_profiler.cpp#L353), which this makes handle correctly.

So I think it's all good.